ci-operator/step-registry/ipi/conf/ipi-conf-ref.yaml

```diff
@@ -28,5 +28,9 @@ ref:
       default:  ""
       documentation: |-
         This field represents how to expose the user-facing endpoints like the Kubernetes API, OpenShift routes of the cluster
+    - name: FEATURE_SET
+      default: ""
```

Comment:
This can have `TechPreviewNoUpgrade` as a default value, and we can avoid updating all the existing jobs.